code/_expts/slotnick_etal_2001/analyze/stats/de_StaticizerCC.m

```diff
@@ -15,9 +15,19 @@
         tmp   = de_calcPErr( vertcat(o.(ds)), mSets.data.test.T, 2);
         
         if (strcmp(mSets.data.taskType, 'categorical'))
-            trial_types = {'on', 'off'};
+            switch(mSets.data.stimSet)
+                case 'blob-dot',
+                    trial_types = {'on', 'off'};
+                otherwise, error('Unimplemented stimSet for categorical. Choose from {blob-dot}');
+            end
         else
-            trial_types = {'near', 'far'};
+            switch(mSets.data.stimSet)
+                case 'blob-dot',
+                    trial_types = {'near', 'far'};
+                case 'paired-squares',
+                    trial_types = {'same', 'different'};
```

Comment:
This is OK, but better if they can be pulled from `mSets.data.test.TLBL` (so that they're not defined here and in the `de_StimCreate` code)